src/app/validation/commandValidation.ts

```diff
@@ -315,7 +315,10 @@ const complete = async (request: CompleteRequest): Promise<void> => {
             'Check your previous DMs from bountybot for the correct id.')
     }
 
-    if (dbBountyResult.status && dbBountyResult.status !== BountyStatus.in_review) {
+    const inReview = dbBountyResult.status && dbBountyResult.status === BountyStatus.in_review;
+    const inProgress = dbBountyResult.status && dbBountyResult.status === BountyStatus.in_progress;
+
```

Comment:
What happens in the vase of !dbBountyResult.status

Reply:
Runtime error handling - if status isn't set, there's something seriously wrong